### ISSUE TITLE
Feature request: button to set container for all imported, restored, or new tabs

### ISSUE THREAD

**Issue by piecevcake** [mozilla/multi-account-containers#1430]
I and thousands of other users import bulk tabs regularly if not daily.  From tree tabs saved groups, which restores a TREE structure of tabs in sidebar (also from Onetab, bookmarks, etc).

When saved they are in containers.  But none of these facilities allows restoration into the original containers.

**With Tree tabs, "reopen tab in container" loses the tree structure, so the backup facility is virtually useless if containers are used.** With bookmarks and onetab it is just an added unnecessary step.

So my workaround would be to save groups by container, then restore each group into the desired container.

If you could provide a button in the dropdown containers list to mark a container for all new, orphan tabs please.

(Naturally it should NOT override the current default of opening _links_ in the parent tab container).

**Note this is a MAJOR security issue, as always opening imported tabs in default container leads to inadvertent logins in wrong containers thence leakage - the very problem containers was supposed to avert.

![ff containers- option A-button to set default container ■ New Issue · mozilla_multi-account-containers @2019 05 30, 11_28_56](https://user-images.githubusercontent.com/32110576/58598089-f19d4300-82ce-11e9-8981-a0230895c4f7.png)

![ff containers- option B-button to set default container ■ New Issue · mozilla_multi-account-containers @2019 05 30, 11_28_56](https://user-images.githubusercontent.com/32110576/58598093-f530ca00-82ce-11e9-83be-eb9bc5658dde.png)

I have raised a request to enable addon developers to import tabs in original containers.  This feature would be _additional_ to that, as users may not always want to open in original containers (or original containers may be unavailable, eg when importing into different profiles).

**Comment by lhorace** [comment]
> If you could provide a button in the dropdown containers list to mark a container for all new, orphan tabs please.

Possible duplicate https://github.com/mozilla/multi-account-containers/issues/914

**Comment by piecevcake** [comment]
NOT a duplicate.  

 #914 refers to the opening of _links_  and requests change to the default behaviour I am happy with for links. 

(There is now an option to open individual links in different container so  #914 is basically solved.)

**Comment by piecevcake** [comment]
> 
> 
> > If you could provide a button in the dropdown containers list to mark a container for all new, orphan tabs please.
> 
> Possible duplicate #914
-----------
NOT a duplicate.

#914 refers to the opening of links and requests change to the default behaviour I am happy with for links.

(There is now an option to open individual links in different container so #914 is basically solved.)

**Comment by lhorace** [comment]
Right, though I think it's all encompassing, anyway see #1343, #1318, but #847 might be what you looking for.

**Comment by piecevcake** [comment]
How long would it take to actually DO it, compared with the time spent on this endless reshuffling????
